ebos/ecloutputblackoilmodule.hh

```diff
@@ -1210,22 +1211,40 @@ public:
                                 tmp_values[8] = get("WWPR")/get("WGPR"); //WellProdDataType::WaterGasRatio                                      
                                 tmp_values[9] = get("WBHP"); //WellProdDataType::BHP
                                 tmp_values[10] = get("WTHP"); //WellProdDataType::THP
-                                //tmp_values[11] = 0; //WellProdDataType::SteadyStatePI //                      
                                 
-                                outputProductionReport_(tmp_values, tmp_names, forceDisableProdOutput);         
+                                const auto prefPhase = well.getPreferredPhase();
+                                auto ftype = [](const auto wtype) -> std::string
+                                {
+                                        switch (wtype) {
+                                        case Phase::OIL:   return "Oil";
+                                        case Phase::WATER: return "Wat";
+                                        case Phase::GAS:   return "Gas";          
+                                        default:
+                                        {
+                                                return "";
+                                        }
+                                        }
+                                };
+                                const auto flowtype = ftype(prefPhase);
+                                Scalar flowPot = 0.0;                                
+                                if(flowtype == "Oil"){ flowPot = tmp_values[2]; } 
```

Comment:
Nitpick: put a space after `if`, and put the code in the blocks (here and the lines below) on separate lines:
```
if (x) {
    // code
} else if (y) {
    // other code
}
```